backend/test/pe-proxy.test.ts

```diff
@@ -0,0 +1,46 @@
+import * as request from 'supertest';
+import app from '../src/api/app';
+import { createUserToken } from './util';
+import { connectToDatabase, UserType } from '../src/models';
+
+describe('pe-proxy', () => {
```

Comment:
two options
- Set PE_API_URL to something like `localhost` (https://github.com/cisagov/crossfeed/blob/31db67469e65b58cc865c004d181b35adf922534/backend/jest.config.js#L15)
- create a mock for createProxyMiddleware that just returns a 200